### ISSUE TITLE
Carthage update failing

### ISSUE THREAD

**Issue by AndrewSB** [layerhq/Atlas-iOS#1091]
`carthage update` is failing currently while pointing at `master`, at `1.0.18`, `1.0.17`. `1.0.16`, and `1.0.15` (1.0.15 doesn't seem to be carthage compatible)

Possibly related to #1088 
Here's my output

```
*** Building scheme "Atlas" in Atlas.xcworkspace
** BUILD FAILED **


The following build commands failed:
	CompileC /Users/asb/Library/Developer/Xcode/DerivedData/Atlas-eoicogtuebmwcnfthkvxzolsxyih/Build/Intermediates/Atlas.build/Release-iphoneos/Atlas.build/Objects-normal/armv7/ATLAddressBarTextView.o Code/Views/ATLAddressBarTextView.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
	CompileC /Users/asb/Library/Developer/Xcode/DerivedData/Atlas-eoicogtuebmwcnfthkvxzolsxyih/Build/Intermediates/Atlas.build/Release-iphoneos/Atlas.build/Objects-normal/armv7/ATLTypingIndicatorViewController.o Code/Controllers/ATLTypingIndicatorViewController.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(2 failures)
/Users/asb/Developer/Real/Carthage/Checkouts/Atlas-iOS/Code/Utilities/ATLMessagingUtilities.h:22:9: fatal error: module 'LayerKit' not found
/Users/asb/Developer/Real/Carthage/Checkouts/Atlas-iOS/Code/Utilities/ATLMessagingUtilities.h:22:9: fatal error: module 'LayerKit' not found
A shell task failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	CompileC /Users/asb/Library/Developer/Xcode/DerivedData/Atlas-eoicogtuebmwcnfthkvxzolsxyih/Build/Intermediates/Atlas.build/Release-iphoneos/Atlas.build/Objects-normal/armv7/ATLAddressBarTextView.o Code/Views/ATLAddressBarTextView.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
	CompileC /Users/asb/Library/Developer/Xcode/DerivedData/Atlas-eoicogtuebmwcnfthkvxzolsxyih/Build/Intermediates/Atlas.build/Release-iphoneos/Atlas.build/Objects-normal/armv7/ATLTypingIndicatorViewController.o Code/Controllers/ATLTypingIndicatorViewController.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(2 failures)
```

**Comment by LukeDurrant** [comment]
+1
The Atlas Framework doesn't currently build looks like
ATLBaseCollectionViewCell was removed from the compiled sources

**Comment by AndrewSB** [comment]
@layerhq would love to get this resolved

**Comment by blakewatters** [comment]
We are prepping to release 0.1.19 which will fix it. In the meantime you can build off a SHA in your `Cartfile` like so:

```
github "layerhq/Atlas-iOS" "0b22fb131a8accb0b1b09e7d2bc07047fad6a51c"
```

Sorry for the turbulence. We need to rev our CI and release scripts to coverage Carthage more cleanly as its higher maintenance than CocoaPods

**Comment by kmahal** [comment]
This fix has been released in `1.0.19`.
